src/view/reviewCommentController.ts

```diff
@@ -124,7 +124,7 @@ export class ReviewCommentController implements vscode.Disposable, CommentHandle
 	}
 
 	async initializeWorkspaceCommentThreads(): Promise<void[]> {
-		await this._prManager.validateDraftMode(this._prManager.activePullRequest!);
+		await this._prManager.validateDraftMode(this._prManager.activeItem!);
```

Comment:
I think this should still be `activePullRequest`. The comment controller here is specifically for review comments, those on files, which issues wouldn't have.